### PR TITLE
refactor: access resources through properties

### LINKED ISSUE #20: Change style to a something more similar to other clients
This API client uses a different style than others without any reason:

JavaScript
```javascript
client.resource('wallets').all()
```
Python
```python
client.wallets.all()
```
Ruby
```ruby
client.wallets.all
```
Java
```java
connection.api().wallets.all();
```

Before releasing the major version (https://github.com/ArkEcosystem/javascript-client/issues/10), this should be changed to be more coherent and reduce boilerplate:
```javascript
client.wallets.all()
```

### PR DESCRIPTION
**Pull request by Pedro-Souza**
fix #20 

now you can call ``` client.wallets.all()```

### CODE REVIEW

**Comment by Pedro-Souza** [comment]
@j-a-m-l Done!

File: lib/index.js [hunk 1]
```diff
@@ -190,13 +200,14 @@ module.exports = class ApiClient {
...
 
-  /**
-   * Create an instance of a version specific resource.
-   * @param  {String}   name
-   * @return {Resource}
-   */
-  resource (name) {
-    //return new resources[`v${this.version}`][name](this.http)
-    return new resources[`api`][name](this.http)
+  setPropertys () {
```
Comment (faustbrian) [inline lib/index.js]: `setProperties`

File: lib/index.js [hunk 2]
```diff
@@ -155,6 +164,7 @@ module.exports = class ApiClient {
   constructor (host, version) {
     this.setConnection(host)
     this.setVersion(version || 2)
+    this.setPropertys()
```
Comment (faustbrian) [inline lib/index.js]: `setProperties `

File: lib/index.js [hunk 3]
```diff
@@ -1,6 +1,16 @@
 const shuffle = require('lodash.shuffle')
 const HttpClient = require('./http')
-const resources = require('./resources')
+/* const resources = require('./resources') */
```
Comment (faustbrian) [inline lib/index.js]: Remove if not used.

**Comment by Pedro-Souza** [comment]
@faustbrian Done!

**Pull request merged**